<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>抽湿机</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>440</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>设备信息</string>
   </property>
   <widget class="QPushButton" name="BtnSearch">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>401</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelStatus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnConnect">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnDisconnect">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupOP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>621</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>操作控制</string>
   </property>
   <widget class="QPushButton" name="BtnPwr">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>电源</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedM1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>60%</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedPwr">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnMode">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>水满</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>除湿</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnHumidity">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedFull">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedH1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>连续</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedH2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>干衣</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedH4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedH3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>静音</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedM2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>70%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>电源</string>
    </property>
   </widget>
   <widget class="QLabel" name="LedM3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
